src/go/rpk/pkg/cli/cmd/debug/bundle/bundle.go

```diff
@@ -158,9 +161,15 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 	command.Flags().DurationVar(
 		&timeout,
 		"timeout",
-		10*time.Second,
+		15*time.Second,
```

Comment:
This should probably be a few seconds larger than the metrics interval

Reply:
In theory, yes. 

In practice, this timeout only stops the context of a few sub-commands. I'll create a ticket so we use this everywhere and make this a few seconds larger than the metrics interval :+1: